IceCreamShop.playground/Contents.swift

```diff
@@ -1 +1,144 @@
+//1. Create a struct called `Flavor`. Add the following properties: Name, Rating Think about what type they should be.
+
+struct Flavor {
+    let name: String
+    let rating: Int
+}
+
+//2. Create an enum called `Size`.
+// Give it a case for small, medium, and large.
+// Make the enum have `Double` raw values. The raw value should represent the size's price. For example, small might be 3.99.
+
+enum Size: Double {
+    case small
+    case medium
+    case large
+}
+
+//3. Create a struct called `Cone`.
+//  Give the struct properties (constants) that hold a flavor, topping (as a `String`), and size.
+//  Create a function called `eat`. It shouldn't have any arguments. It should print a string of text that includes the name of the cone's flavor. For example: "Mmm! I love <flavor here>!"
+
+struct Cone {
+    let flavor: String
+    let topping: String
+    let size: Size
+}
+
+let myCone = Cone(flavor: "Chocolate",
+                  topping: "hotFudge",
+                  size: .large)                  //my order
+
+
+func eat(Flavor: String) {
```

Comment:
"Create a function called 'eat'. It shouldn't have any arguments"
Here you gave it an argument of "Flavor" of type "String"